router/src/main/java/com/flownav/router/navcomp/FlowNavFragmentNavigation.kt

```diff
@@ -59,9 +61,9 @@ fun FlowNavFragmentRouter.workWithNavGraphOf(
 
 fun FlowNavFragmentRouter.navigateTo(destination: String, lifecycleOwner: LifecycleOwner) {
     getNavControllerByLifecycle(lifecycleOwner)?.apply {
-        FlowNavApp.getFragmentMap()[destination]?.run {
-            this@apply.navigate(fragmentId)
-        }
+        FlowNavApp.getEntryMap()[destination]
+            ?.takeIf { it.actionId != -1 }
```

Comment:
acho que da para criar uma extension ou alguma função com essa verificação, pode terminar ficar varias verificações no projeto todo com `action != -1` e ficar confuso depois de um tempo, acho que poderia ter uma função ou extension to tipo, `entryConfig.isDefaultId()`

Reply:
Acho válido!